src/rard/research/models/original_text.py

```diff
@@ -19,28 +20,47 @@ def related_lock_object(self):
     class Meta:
         ordering = ("citing_work", "reference_order")
 
+    @property
+    def reference(self):
+        references = Reference.objects.filter(original_text=self)
+        if references:
+            # only return the number values as the editor isn't critical
+            return references.first().reference_position
+        else:
+            return ""
```

Comment:
We can get rid of this now right? Or are there templates using it somewhere?

Reply:
I think it's good to go; the forms shouldn't be affected